### PR TITLE
Add flags/arguments order verification to one-arg commands - easy

### PR DESCRIPTION
**Pull request by boaz1337**
A follow up to https://github.com/projectatomic/buildah/pull/968#issuecomment-417158706

It's a work in progress as I have to add more tests and make sure I covered all the necessary commands.

### CODE REVIEW

File: pkg/cli/common.go [hunk 1]
```diff
@@ -280,3 +281,12 @@ func DefaultIsolation() string {
 	}
 	return buildah.OCI
 }
+
+func VerifyFlagsArgsOrder(args []string) error {
+	for _, arg := range args {
+		if strings.HasPrefix(arg, "-") {
+			return errors.Errorf("%s should be set before the image name", arg)
```
Comment (rhatdan) [inline pkg/cli/common.go]: Perhaps make this a little more informative.

"No options (%s) can be specified after the image or container name"

Comment (boaz1337) [reply pkg/cli/common.go]: Yep, I like this better. Thanks!

**Comment by boaz1337** [comment]
This PR is trivial: adding verification to only one argument commands such as `commit`, `config` and etc.
The next part is adding this verification to `images`, `mount`, `push`, `rename` and etc.

Commands that don't need this verification are `version` and `containers`.

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**